File: linear_search.c
#include <stdio.h>

int linearSearch(int arr[], int size, int target) {
	int i;
    for (i = 0; i < size; i++) {
        if (arr[i] == target) {
            return i;
        }
    }
    return -1;
}

int main() {
    int numbers[] = {10, 20, 30, 40, 50};
    int size = sizeof(numbers) / sizeof(numbers[0]);
    int searchTarget = 30;

    int result = linearSearch(numbers, size, searchTarget);

    if (result != -1) {
        printf("Element %d found at index %d.\n", searchTarget, result);
    } else {
        printf("Element %d not found in the array.\n", searchTarget);
    }

    return 0;
}

File: linearsearch.py
def linear_search(arr, target):
    for index in range(len(arr)):
        if arr[index] == target:
            return index  
    return -1  

my_list = [10, 23, 45, 70, 11, 15]
search_target = 70

result = linear_search(my_list, search_target)

if result != -1:
    print(f"Element {search_target} found at index: {result}")
else:
    print(f"Element {search_target} not found in the list.")

search_target_not_present = 99
result_not_present = linear_search(my_list, search_target_not_present)

if result_not_present != -1:
    print(f"Element {search_target_not_present} found at index: {result_not_present}")
else:
    print(f"Element {search_target_not_present} not found in the list.")